source/what-is-the-bi-connector.txt

```diff
@@ -31,60 +35,57 @@ The |bi| acts as a layer that translates queries and data between a
 business intelligence tools.
 
-.. figure:: /images/bi-connector/components-all.jpg
-   :figwidth: 700px
-   :align: center
-   :alt: Diagram showing that other BI tools communicate with the DSN, which communicates with MongoDB's BI Connector, which in turn communicates with the MongoDB database.
-
 System Components
 -----------------
 
 A complete BI system includes the following components:
 
-MongoDB database
-  Data storage.
+BI Tool
+  Retrieve, visualize, and report on data from a database.
+
+|jdbc| or |odbc| drivers
+  Provides a standard method to connect to the |bi-short| and MongoDB
+  databases from a BI Tool. Depending on what your BI Tool supports, you
+  can use |jdbc| or |odbc| drivers to connect to the |bi-short|.
 
 BI Connector
   Provides a relational schema and translates |sql|
   queries between your BI tool and MongoDB.
 
-|jdbc| and |odbc| drivers
-  Provides a standard method to connect to the |bi-short| and MongoDB
-  databases from a BI Tool. Depending on what your BI Tool supports, you
-  can use |jdbc| and |odbc| drivers to connect to the |bi-short|.
+MongoDB database
+  Data storage.
 
-BI Tool
-  Retrieve, visualize, and report on data from a database.
+.. figure:: /images/bi-connector/components-all.jpg
+   :figwidth: 700px
+   :align: center
+   :alt: Diagram showing that other BI tools communicate with the DSN, which communicates with MongoDB's BI Connector, which in turn communicates with the MongoDB database.
+
+Your **BI Tool** and **Driver** typically run on your local machine. 
+Your **{+bi-short+} instance** and your **database** can run on your 
+local machine, in a data center, or hosted in the cloud.
 
-Your BI Tool and Driver typically run on your local machine. Your
-database and your |bi-short| instance can run on your local machine,
-in a data center, or hosted in the cloud.
+|bi-short| Configurations
+-------------------------
 
 The following sections describe possible configurations for a
-BI system.
+BI system:
 
 Hosted Database and |bi-short|
-------------------------------
-
-.. note::
-
-   If you don't want to install and manage the |bi-short|, you can use
-   {+atlas+}, our fully-managed database-as-a-service. Databases hosted
-   on {+atlas+} natively support the |bi| and make it easy to start
-   analyzing your data with your preferred BI tools. To get started,
-   :website:`create an {+atlas+} cluster </cloud/atlas/lp/general>`.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-In this scenario, the database and |bi-short| applications both run on
-{+atlas+}, and you set up your :doc:`DSN </tutorial/create-system-dsn>`
-with connection information that {+atlas+} provides.
+In this configuration, the database and |bi-short| applications both 
+run on {+atlas+}, MongoDB's fully-managed database-as-a-service. 
+Databases hosted on {+atlas+} natively support the |bi-short|. 
+{+atlas+} provides the connection information for your |jdbc| or 
+|odbc| driver.
```

Comment:
Do you think we should link to the doc or specify where the connection information is provided?

Reply:
Yes, but I need to create a driver overview page to link to first - right now that info is weirdly scattered across [Create a System DSN](https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/docsworker-xlarge/DOCSP-18336/tutorial/create-system-dsn/) (for ODBC) and the sparse [JDBC Driver](https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/docsworker-xlarge/DOCSP-18336/reference/jdbc-driver/) page.